pkg/client/client.go

```diff
@@ -10,9 +10,19 @@ import (
 )
 
 type (
+	// Options is the struct describing additional information pertinent to an agent that isn't a data reading
+	// These fields will then be uploaded together with data readings.
+	Options struct {
+		OrgID              string
+		ClusterID          string
+		ClusterName        string
```

Comment:
What's the difference between ClusterName and ClusterID? It seems like ClusterName is set to ClusterID in `run.go`:

```go
err := preflightClient.PostDataReadingsWithOptions(readings, client.Options{
	ClusterName:        config.ClusterID,
	ClusterDescription: config.ClusterDescription,
})
```

and ClusterID isn't set?

Reply:
Looking up earlier in the file run.go:

```go
// orgID and clusterID are not required for Venafi Cloud auth
```

Sounds like we could comment the `Options` struct like this:

```go
	Options struct {
		OrgID              string // Only used with platform.jetstack.io
		ClusterID          string // Only used with platform.jetstack.io
		ClusterName        string // Only used with Venafi Cloud
		ClusterDescription string // Only used with Venafi Cloud
	}
```